projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.test.ts

```diff
@@ -167,6 +167,6 @@ describe('Timeline Card List component', () => {
     });
 
     expect(spectator.query('.time')).not.toExist();
-    expect(spectator.query('.button')).not.toBe(spectator.query('.with-margin'));
+    expect(spectator.query('.button')).toBe(spectator.query('.with-margin'));
```

Comment:
this is likely not a useful thing to verify any more since it's just a static class.

Reply:
We do want to ensure margin is applied. But yeah, we dont test that usually